pages/k8s/quickstart.md

```diff
@@ -0,0 +1,265 @@
+---
+title: "Quickstart | Canonical Distribution of Kubernetes&reg;"
+keywords: quickstart
+tags: [getting_started]
+sidebar: k8smain-sidebar
+permalink: quickstart.html
+layout: base
+toc: False
+summary: This page is currently a work in progress. For existing documentation, please visit <a href="https://kubernetes.io/docs/getting-started-guides/ubuntu/"> https://kubernetes.io/docs/getting-started-guides/ubuntu/ </a>
+---
+<style>
+img {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 75%;
+  height: auto;
+}
+</style>
+
+<div class="p-content__row">
+    <div class="u-equal-height">
+        <div class="col-6">
+
+<h2> Quick Start</h2>
+
+<p>The Canonical Distribution of Kubernetes<sup>&reg;</sup> delivers a ‘pure K8s’
+experience, tested across a wide range of clouds and integrated with modern
+metrics and monitoring. It works across all major public clouds and private
+infrastructure, enabling your teams to operate Kubernetes clusters on demand,
+anywhere.</p>
+
+<p>With this quick start guide and some tools from Canonical, you can have all
+this distilled Kubernetes expertise operating in your own  cluster
+running on a cloud of your choice in minutes!</p>
```

Comment:
This sentence seems a little wordy and awkward, I wonder if we could simplify. One idea:

"With this quick start guide and some tools from Canonical, you'll have a Kubernetes cluster running on the cloud of your choice in minutes!"